docs/content/tools/collector/_index.md

```diff
@@ -0,0 +1,494 @@
+$iswindows ? $(mkdir C:\WARA -ErrorAction SilentlyContinue;cd C:\WARA) : (Write-Host "C:\WARA - Not Required")
+
+#Download the latest version of the script
+invoke-webrequest https://aka.ms/aprl/tools/1 -out 1_wara_collector.ps1
+
+#Remove file blocking if active and running windows
+$iswindows ? (unblock-file ./1_wara_collector.ps1) : (Write-host "Unblock not required - Not Windows OS")
+
+#Modify these parameters and run the script
+./1_wara_collector.ps1 -TenantID "00000000-0000-0000-0000-000000000000" -SubscriptionIds "/subscriptions/00000000-0000-0000-0000-000000000000"
+```
+### Local Machine
+
+![Alt text](quickstartexample.gif)
+
+### Cloud Shell
+
+![Alt text](quickstartexample_cloudshell.gif)
+
+## How to download
+
+- [GitHub Link to Download](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/blob/main/tools/1_wara_collector.ps1)
+- Download the script using command-line
+    ```shell
+    iwr https://aka.ms/aprl/tools/1 -out 1_wara_collector.ps1
+    ```
+- [GitHub Link to Sample Output](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/blob/main/tools/sample-output/WARA_File_2024-05-07_11_59.json)
+
+## How to run the script
+
+See at the end of this page various examples of how to run this script - [Examples](#how-to-run-the-1_wara_collectorps1)
+
+**You have two options to run the collector script:**
+
+1. Cloud Shell - Requires Cloud Shell be configured with write access to a file share within the same tenant
+
+2. Local Machine - Requires current modules leveraged in the script be installed
+
+### 1.1 - Cloud Shell
+
+1. From the [Azure Portal](https://portal.azure.com/) open Cloud Shell, select PowerShell instead of BASH
+
+    - If this is your first time using Cloud Shell, refer to the getting started guide from Microsoft Learn - [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/get-started/classic?tabs=azurecli#start-cloud-shell).
+
+    {{< figure src="../../img/tools/collector-1.png" width="100%" >}}
+
+2. Upload the WARA Collector Script to Cloud Shell
+  {{< figure src="../../img/tools/collector-2.png" width="60%" >}}
+
+    Or download the script from GitHub
+
+    ```shell
+    iwr https://aka.ms/aprl/tools/1 -out 1_wara_collector.ps1
+    ```
+
+3. Execute script leveraging parameters
+   - The script accepts both short and/or full paths.
+
+    {{< figure src="../../img/tools/collector-3.png" width="100%" >}}
+
+    For complex Subscription, ResourceGroups and Tags filtering scenarios we highly recommend using [ConfigFiles - See here an example under config.txt in Example 5](#example-5)
+
+
+
+4. Select "A" to allow modules to install
+  {{< figure src="../../img/tools/collector-4.png" width="100%" >}}
+
+1. After Script completes, download the produced json file to your machine
+  {{< figure src="../../img/tools/collector-5.png" width="100%" >}}
+
+1. Read how to execute the [Analyzer Script](../analyzer/_index.md) to generate the Excel report.
```

Comment:
```suggestion
1. From the [Azure Portal](https://portal.azure.com/) open Cloud Shell, select PowerShell instead of BASH

    - If this is your first time using Cloud Shell, refer to the getting started guide from Microsoft Learn - [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/get-started/classic?tabs=azurecli#start-cloud-shell).

    {{< figure src="../../img/tools/collector-1.png" width="100%" >}}

1. Upload the WARA Collector Script to Cloud Shell
  {{< figure src="../../img/tools/collector-2.png" width="60%" >}}

    Or download the script from GitHub

    ```shell
    iwr https://aka.ms/aprl/tools/1 -out 1_wara_collector.ps1
    ```

1. Execute script leveraging parameters
   - The script accepts both short and/or full paths.

    {{< figure src="../../img/tools/collector-3.png" width="100%" >}}

    For complex Subscription, ResourceGroups and Tags filtering scenarios we highly recommend using [ConfigFiles - See here an example under config.txt in Example 5](#example-5)



1. Select "A" to allow modules to install
  {{< figure src="../../img/tools/collector-4.png" width="100%" >}}

1. After Script completes, download the produced json file to your machine
  {{< figure src="../../img/tools/collector-5.png" width="100%" >}}

1. Read how to execute the [Analyzer Script](../analyzer/_index.md) to generate the Excel report.
```